salt/modules/mssql.py

```diff
@@ -250,23 +300,97 @@ def login_exists(login, **kwargs):
         return 'Could not find the login: {0}'.format(e)
 
 
-def user_exists(username, **kwargs):
+def login_create(login, new_login_password=None, new_login_domain=None, new_login_roles=None, new_login_options=None, **kwargs):
```

Comment:
This is a diff issue.
`login_create` is a new function
`user_exists` is right after this function